<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="harshan-VivoBook-15-ASUS-Laptop-X507UAR" failures="3" tests="4" name="Test" time="47.099" errors="0" timestamp="2023-09-24T23:42:35 IST">
  <testcase classname="com.inet.testCase.TC_loginTest_002" name="LoginDDT" time="11.361"/>
  <testcase classname="com.inet.testCase.TC_loginTest_002" name="LoginDDT" time="6.345">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.inet.testCase.TC_loginTest_002.LoginDDT(TC_loginTest_002.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
  <testcase classname="com.inet.testCase.TC_loginTest_002" name="LoginDDT" time="5.9">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.inet.testCase.TC_loginTest_002.LoginDDT(TC_loginTest_002.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
  <testcase classname="com.inet.testCase.TC_loginTest_002" name="LoginDDT" time="6.029">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.inet.testCase.TC_loginTest_002.LoginDDT(TC_loginTest_002.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
</testsuite> <!-- Test -->
